eduNEXT/tutor-contrib-ontask
# feat: add ontask service definitions for dev/local

### Description
This PR adds the necessary definitions for the OnTask service: docker-compose files, initialization tasks, configurations, and variables. 

### How to test
1. Install and enable this plugin in your tutor environment using the `MJG/ondata-service` branch
2. Initialize services: `tutor dev|local do init --limit ontask`
3. Run services: `tutor dev|local start -d ontask`
4. Check your OnTask installation: `ontask.<LMS DOMAIN>` 
![image](https://github.com/eduNEXT/tutor-contrib-ontask/assets/64440265/932197d6-c79e-48a1-9e28-4a4f3b0bce06)
